console/console-init/ui/mock-console-server/mock-console-server.js

```diff
@@ -23,7 +23,7 @@ function calcLowerUpper(offset, first, len) {
   if (first !== undefined && first > 0) {
     upper = Math.min(lower + first, len);
   }
-  return {lower, upper};
+  return { lower, upper };
```

Comment:
Combining the reformatting of the file with functional change is unhelpful to reviewers.  Can you revert the formatting changes?  If you want to reformat pls. submit it as a separate commit.,